compiler/src/org.graalvm.compiler.java/src/org/graalvm/compiler/java/LambdaUtils.java

```diff
@@ -51,6 +51,8 @@
 public final class LambdaUtils {
     private static final Pattern LAMBDA_PATTERN = Pattern.compile("\\$\\$Lambda\\$\\d+[/\\.][^/]+;");
     private static final char[] HEX = "0123456789abcdef".toCharArray();
+    public static final String SPLIT_BY_LAMBDA = "\\$\\$Lambda\\$";
+    public static final String LAMBDA_FUNCTIONS_NAME_PATTERN = "$$Lambda$";
```

Comment:
LAMBDA_CLASS_NAME_PATTERN is maybe more accurate.

Reply:
Why don't we use the `LAMBDA_PATTERN` above?

Reply:
> LAMBDA_CLASS_NAME_PATTERN is maybe more accurate.

Changed as suggested.

Reply:
> Why don't we use the `LAMBDA_PATTERN` above?

I expects `;` at the end (not optional) which is not case in cases which are important for this PR. So, we need 2 new constants in LambdaUtils class.